src/transforms/throttle.rs

```diff
@@ -17,24 +18,27 @@ use crate::{
 };
 
 /// Configuration for the `throttle` transform.
+#[serde_as]
 #[configurable_component(transform("throttle"))]
 #[derive(Clone, Debug, Default)]
-#[serde(deny_unknown_fields, default)]
```

Comment:
Telling `serde` to use a default value for deserializing the configuration of a typed component makes no sense, because these types are never a fixed field... only part of a field using their containing enums i.e. `Transforms`.

It does, however, lead to the `vector-config-macros` code thinking that there's a default value that should be added to the configuration schema, which again... does not make sense at _this_ particular level.